Ginger/GingerCoreNET/Drivers/CommonLib/FriendlyLocatorlist.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using OpenQA.Selenium;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace GingerCoreNET.Drivers.CommonLib
+{
+    public class FriendlyLocatorElement
+    {
+        public string position { get; set; }
```

Comment:
is this class and object being used anywhere?
if it is needed then change string position to enum

Reply:
Done